semantic_convention/generate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+rm -rf opentelemetry-specification || true
+git clone https://github.com/open-telemetry/opentelemetry-specification.git --depth 1
```

Comment:
We should pin this to a certain commit, or check out a release branch to have a latest 0.x version at least.